ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -414,6 +414,9 @@ interface {{ ethernet_interface }}
               ethernet_interfaces[ethernet_interface].bfd.multiplier is arista.avd.defined %}
    bfd interval {{ ethernet_interfaces[ethernet_interface].bfd.interval }} min-rx {{ ethernet_interfaces[ethernet_interface].bfd.min_rx }} multiplier {{ ethernet_interfaces[ethernet_interface].bfd.multiplier }}
 {%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].bfd.echo is arista.avd.defined %}
```

Comment:
Should check:
```ethernet_interfaces[ethernet_interface].bfd.echo is arista.avd.defined(true)```
Otherwise you will render the command also when it is defined as false and this conflicts with documentation

Reply:
it is the documentation that is wrong. I commented above

Reply:
Great feedback!  Both items are now fixed.  Thanks!